content/post/azure-monitor-terraform.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Azure Monitor & Terraform: Collecting custom log files"
+date: 2023-10-03T14:46:23+01:00
+hero: /images/hero-3.jpg
+excerpt: "Working with Azure Monitor in collecting custom log files using Terraform"
+timeToRead: 5
+authors:
+  - Dylan McGinley
+draft: true
+license: cc-by
+---
```

Comment:
Ah, if the article exists somewhere else (Medium?), the frontmatter should have the canonical link URL.